easybuild/easyconfigs/s/setuptools/setuptools-41.0.1.eb

```diff
@@ -0,0 +1,25 @@
+easyblock = "PythonPackage"
+
+name = "setuptools"
+version = "41.0.1"
+
+homepage = "https://pypi.org/project/setuptools"
+description = """Easily download, build, install, upgrade, and uninstall Python packages"""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_ZIP]
+checksums = ['a222d126f5471598053c9a77f4b5d4f26eaa1f150ad6e01dcf1a42e185d05613']
+
+download_dep_fail = True
+use_pip = False
+
+req_py_majver = 3
```

Comment:
Please add a comment to clarify why you do this (it makes sense, but it'll look weird when looking at this outside of the ReFrame context).

It also makes sense to use a versionsuffix like `-py3` here imho...

Reply:
> It also makes sense to use a versionsuffix like -py3 here imho...

I had the same feeling, but I was expecting your feedback. :-)